plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/WooCommercePayments.php

```diff
@@ -179,11 +180,11 @@ public static function is_account_partially_onboarded() {
 	 * @return bool
 	 */
 	public static function is_supported() {
-		$suggestions              = Suggestions::get_suggestions();
+		$suggestions              = Suggestions::get_suggestions( DefaultPaymentGateways::get_all() );
```

Comment:
@chihsuan I realized that this new call now skips the filter `woocommerce_admin_payment_gateway_suggestion_specs` that gets [called here](https://github.com/woocommerce/woocommerce/blob/5c980cc6299af86ed18bb8cb10695364467db761/plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/Init.php#L76).

This could be problematic with bridge where the filter is used for limiting gateways in free trial plans

Reply:
@ilyasfoo I was aware of that, but I thought it was okay. We just want to update the WooCommerce Payments task to use a static country list, so I didn't add that filter here when I created this PR. 

From my understanding,  The `is_supported` method is used to check if the store is eligible for WooCommerce Payment, and it only affects WooCommerce Payments. I don't think bridge should modify the return value of this method. 🤔 


Do you think it will still be problematic for the bridge?  If so, I can create a new PR to add the filter. 🙏

Reply:
> The is_supported method is used to check if the store is eligible for WooCommerce Payment, and it only affects WooCommerce Payments. I don't think bridge should modify the return value of this method. 🤔

@chihsuan Good point, since this use is limited I think it should be fine then!